flink-table-store-core/src/main/java/org/apache/flink/table/store/file/manifest/ManifestEntry.java

```diff
@@ -38,6 +38,7 @@
     private final int bucket;
     private final int totalBuckets;
     private final SstFileMeta file;
+    private final Identifier identifier;
```

Comment:
I mean you can extract this class as method local variable. No need to be a class member here.

Reply:
`Identifier` is used in multiple classes so we can't.